__tests__/app.test.js

```diff
@@ -134,3 +134,90 @@ describe('GET /api/articles', () => {
         })
     })
 });
+
+describe('GET /api/articles/:article_id/comments', () => {
+    test('status:200, responds with a JSON object containing an array of comment objects for the given article_id of which each object has the following properties: comment_id, votes, created_at, author, body, article_id', () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then(({body}) => {
+
+            const { comments } = body;
+
+            expect(comments).toHaveLength(11);
+
+            comments.forEach((comment) => {
+                expect(comment).toMatchObject({
+
+                    comment_id: expect.any(Number),
+                    votes: expect.any(Number),
+                    created_at: expect.any(String),
+                    author: expect.any(String),
+                    body: expect.any(String),
+                    article_id: expect.any(Number)
+
+                })
+            })
+
+        })
+    })
+    test('status:200, comments are sorted by their created_at date in ascending order', () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then(({body}) => {
+
+            const { comments } = body;
+
+            expect(comments).toHaveLength(11)
+
+            expect(comments).toBeSorted('created_at', { ascending: true })
+
+        })
+    })
+    test('status:200, returns an array containing an article object which contains no comments when article_id is valid but does not have any comments', () => {
+        return request(app)
```

Comment:
this is off behaviour to design. I'm request all the comments associated with an article.
Assuming the article exists, I'm expect an array of all of the comments. If the there were no comments, that array would be empty. (Still a 200 though!)